bee-network/bee-autopeering/src/local/salt.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    proto,
+    time::{self, Timestamp, HOUR},
+};
+
+use ring::rand::{SecureRandom as _, SystemRandom};
+
+use std::{
+    convert::TryInto,
```

Comment:
Can you remove all imports of TryInto/TryFrom please ? As they are now automatically imported with edition 2021.

Reply:
Done